package.json

```diff
@@ -31,7 +32,15 @@
     "typescript": "4.3.5"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "eslint --cache --fix",
-    "*.{js,jsx,ts,tsx,css,md}": "prettier --write"
-  }
+    "*.{js,json,md}": [
+      "prettier --write"
+    ],
+    "**/**.css": [
+      "prettier --write"
+    ],
+    "**/*.{js,jsx,ts,tsx}": [
+      "eslint --fix"
+    ]
```

Comment:
Why are you avoiding prettier from formating the TS files?
I thought ESLint and Prettier should work well side-by-side with the current setup.

Reply:
If we include `eslint-plugin-prettier` in the ESLint config, it should automatically take all config rules + autofixes directly from the Prettier config, without having to go through Prettier.